admin-core/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM eclipse-temurin:11.0.21_9-jdk-nanoserver
+
+RUN mkdir distr
+RUN mkdir work
+
+COPY build/libs/admin-core-0.8.3-all.jar ./app/admin-core-0.8.3-all.jar
+
+EXPOSE 8090
```

Comment:
Allow to customize port values via env variable (make sure to set defaults)
```
# check exact syntax with docs / examples - I'm writing this by memory
ENV MY_PORT=8090
EXPOSE $MY_PORT
```

I suggest to name those API_PORT, DEBUG_PORT, SECURE_API_PORT respectively

Reply:
done